fluss-server/src/main/java/com/alibaba/fluss/server/replica/ReplicaManager.java

```diff
@@ -1499,8 +1499,12 @@ public Replica getReplicaOrException(TableBucket tableBucket) {
         HostedReplica replica = getReplica(tableBucket);
         if (replica instanceof OnlineReplica) {
             return ((OnlineReplica) replica).getReplica();
+        } else if (replica instanceof OfflineReplica) {
+            throw new StorageException(tableBucket + " is offline");
```

Comment:
```suggestion
            throw new StorageException(tableBucket + " is offline on server " + serverId);
```

Add server id information?